src/Aeon.Acquisition/CheckDirtyRepo.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Bonsai;
+using System;
+using System.ComponentModel;
+using System.Reactive.Linq;
+using LibGit2Sharp;
+
+namespace Aeon.Acquisition
+{
+
+    [Combinator]
+    [Description("From a Repository object determines if the repository is dirty or uncommited changes exist")]
+    [WorkflowElementCategory(ElementCategory.Transform)]
+
+    public class CheckDirtyRepo
```

Comment:
Best to use `Has` or `Is` prefix instead of `Check` for conditional tests.

Also, can we rewrite the operator in the positive and call it i.e. `IsRepositoryClean`? The reason is that the operator checks not only whether the repository is dirty but also if it doesn't have any untracked files (which technically is a different status).

Checking online precedents it seems the most commonly used way to summarize this is to ask whether the repository is "clean".

Reply:
makes sense!